app/dashboard/organisation/[id]/proxy/page.tsx

```
@@ -0,0 +1,5 @@
+const Proxy = () => {
```

Biome (1.9.4)

[error] Do not shadow the global "Proxy" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)